src/pages/ProposalDetails/VotesBlock/AgainstVotes/AgainstVotes.component.tsx

```diff
@@ -4,6 +4,15 @@ import { VoteAgainstButton } from '../VotesBlock.voteButtons';
 import { VoteActionBlock } from '../VotesBlock.voteAction';
```

Comment:
two different components (AgainstVotesComponent and ForVotesComponent) are ok, but they are very similar. 
I will create some base component and pass props into it, with `iconSrc`, `type` etc.